### PR TITLE
Add cookie-policy v3.0.3

### LINKED ISSUE #248: Add cookie policy and update GTM
is this required? need to ask KO

## On the site
in package.json
```
"@canonical/cookie-policy": "3.0.2",
```

in templates/partial/_footer.html
```
<a href="" class="js-revoke-cookie-manager">Manage your tracker settings</a>
```

### add the js (depends on site)
static/js/base.js
```
import { cookiePolicy } from "@canonical/cookie-policy";
cookiePolicy();
```
or
```
<script src="/static/js/modules/cookie-policy/cookie-policy.js"></script>
<script src="/static/js/modules/global-nav/global-nav.js"></script>
<script>
  canonicalGlobalNav.createNav({
    maxWidth: '72rem',
  });
  cpNs.cookiePolicy();
</script>
```

### add to scss
static/sass/styles.scss
```
// import cookie policy
@import "@canonical/cookie-policy/build/css/cookie-policy";
```

## On the GTM

1. add variable - `_cookies_accepted`
2. add trigger - `All pages && cookie (all or functionality)` - page view regex match `(all|functionality)`
3.add trigger -  `All pages && cookie (all or performance)` - regex match `(all|performance)`
4. add these to the triggers based on [this sheet](https://docs.google.com/spreadsheets/d/1PQEmbJPEG-76FLtLVOO271615V5aor66UhxfomZF-lo/edit#gid=0}
5. tracker `track pageview for cookie acceptance` for pageview
6. trigger - custom `track pageview event` on event `pageview`
#### performance
* ga
* crazyegg
* usabilla
* vimeo	
* google optimise

#### functionality
* linked in	
* livechat	
* facebook	
* twitter	
* bombora	
* google adwords	
* Marketo Munchkin

### PR DESCRIPTION
**Pull request by anthonydillon**
## Done
Add cookie-policy v3.0.3

**Do not merge until @pmahnke has done the GTM changes**

## QA
- Load the demo and you should see the cookie policy
- Open the manager and select something
- See it is gone
- Click the link in the footer to manage cookies
- See it opens again without errors in the console.

## Issue / Card
Fixes #248

### CODE REVIEW

File: templates/_base/footer.html [hunk 1]
```diff
@@ -11,6 +11,9 @@
         <li class="p-inline-list__item">
           <a class="p-link--soft" href="https://github.com/canonical-websites/jp.ubuntu.com/issues/new" id="report-a-bug">Report a bug on this site</a>
         </li>
+        <li class="p-inline-list__item">
```
Comment (pmahnke) [inline templates/_base/footer.html:14]: I think Bartek was right, this would be better before the 'report a bug'... link


What do you think?

**Review by pmahnke — APPROVED** [review]
+1, with a question

File: templates/_base/footer.html (hunk 1 above)
Comment (pmahnke) [reply templates/_base/footer.html:14]: Also, I am ready with GTM when we release.

Comment (anthonydillon) [reply templates/_base/footer.html:14]: Done.

**Pull request merged**